README.md

```diff
@@ -1 +1,59 @@
-# registry-etcd
\ No newline at end of file
+# registry-etcd
```

Comment:
Please add  (This is a community driven project) right after the title to show that this is a community driven project